runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/control/RemoteExecutionTest.java

```diff
@@ -212,9 +212,12 @@ public void process(ProcessContext ctxt) {
               (FnDataReceiver<? super WindowedValue<?>>) outputContents::add));
     }
     // The impulse example
-    try (ActiveBundle<byte[]> bundle = processor.newBundle(outputReceivers)) {
+
+    try (ActiveBundle<byte[]> bundle =
+        processor.newBundle(outputReceivers, BundleProgressHandler.unsupported())) {
```

Comment:
In all your tests where you used BundleProgressHandler.unsupported(), shouldn't the try-with-resources on close invoke the BundleProgressHandler.onCompleted method and throw an UnsupportedOperationException?

Reply:
onCompleted gets called by ActiveBundle.close(). BPH.unsupported() throws an UOE only if there's a split - to reduce confusion, I renamed it to BPH.defaultInstance().